modules/openapi-generator/src/main/resources/php-symfony/api_controller.mustache

```diff
@@ -60,10 +60,16 @@ class {{controllerName}} extends Controller
         {{#bodyParams}}
         // Make sure that the client is providing something that we can consume
         $consumes = [{{#consumes}}'{{{mediaType}}}'{{#hasMore}}, {{/hasMore}}{{/consumes}}];
-        $inputFormat = $request->headers->has('Content-Type')?$request->headers->get('Content-Type'):$consumes[0];
-        if (!in_array($inputFormat, $consumes)) {
-            // We can't consume the content that the client is sending us
-            return new Response('', 415);
+        if (sizeof($consumes) > 0) {
```

Comment:
Well, PR is ready to be merged now. But I would suggest to put your changes into base `Controller/Controller.php` class method and cover it with unit tests to avoid this bug in future. Public method(maybe static) like:
```php
public static function isContentTypeAllowed(Request $request, array $consumes = []): bool;
```

Of course you can stay with your changes, it's just a suggestion.

Reply:
This is a shame, but I'm not sure that I can correctly cover the code with a unit test :( There is no experience with this yet.